packages/playground/docs/getting-started-nested.md

```diff
@@ -7,32 +7,92 @@ permalink: /playground/
+
+UI5 Web Components are shipped as ES6 modules and published on NPM. If you are already using a framework that handles bundling of ES6 modules, skip forward to [Installation](#installation) or check the Tutorials section.
+
+## Creating a project
+
+In case you don't have an existing project with frontend tooling, you can start a new one using [Vite](https://vite.dev). The only prerequisite is to have [Node.js](https://nodejs.org/) installed.
+
+```console
+npm init @vitejs/app
+```
+
+- Select a project name: `my-webcomponents-app`
+- Select a framework: `vanilla`
+- Select a variant: `JavaScript`
+
+This command will create a project structure under `my-webcomponents-app`. Follow the instructions on the screen to install the initial depenednecies and start a dev server.
+
+```
+cd my-webcomponents-app
+npm install
+npm run dev​
+```
+
+You should see a message that a server is running on `Local: http://localhost:3000/`. Launch your favourite IDE and change some code, the browser page will be automatically refreshed.
 
 ## Installation
+<a name="installation"></a>
 
-### Install ```@ui5/webcomponents```
+Install ```@ui5/webcomponents```
 
 ```bash
-npm install @ui5/webcomponents
+npm install @ui5/webcomponents --save
 ```
 
-## Read the Docs
+This command downloads the published source code of UI5 web components to the `node_modules` folder. Now you can add them to your code to get them on the web page.
+
+## Documentation
 
-Documentation and tutorials about UI5 Web Components can be found [here](./playground/docs/)
+Browse the available UI5 Web Components and check their API in the [playground](./playground/docs/).
 
 ## Usage
 
-### Import the needed Web Component
+### 1. Import the needed Web Component.
 
 ```js
-import "@ui5/webcomponents/dist/Button";
+// src/main.js
+import "@ui5/webcomponents/dist/Button.js";
 ```
 
-### Instantiate the Web Component
+This line runs the code that registers the `<ui5-button>` tag with the browser. Now when the browser sees such a tag, it knows how to render it.
 
-```html
-<ui5-button>Hello world!</ui5-button>
+### 2. Instantiate the Web Component.
+
+Next, change the application markup in the same file to display the button on the page:
+```js
+// src/main.js
+...
+document.querySelector('#app').innerHTML = `
+  <ui5-button>Hello UI5 Web Components</ui5-button>
+`
+```
+
+Check the browser to see the displayed button. You can now use UI5 Web Components just like normal HTML elements - set attributes on them, attach events, and more as descirbed in the documentation.
+
+### 3. Production Build
```

Comment:
To be consistent with the other steps, this one should also start with a verb. For example,
Run a production build.

Reply:
done